ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/SlotProcessorUtil.java

```diff
@@ -15,45 +15,28 @@
 
 import static java.lang.Math.toIntExact;
 import static tech.pegasys.artemis.datastructures.Constants.LATEST_BLOCK_ROOTS_LENGTH;
-import static tech.pegasys.artemis.datastructures.Constants.LATEST_RANDAO_MIXES_LENGTH;
 
 import com.google.common.primitives.UnsignedLong;
 import java.util.List;
 import net.consensys.cava.bytes.Bytes32;
 import tech.pegasys.artemis.datastructures.Constants;
-import tech.pegasys.artemis.datastructures.blocks.BeaconBlock;
 import tech.pegasys.artemis.datastructures.state.BeaconState;
 import tech.pegasys.artemis.datastructures.util.BeaconStateUtil;
 import tech.pegasys.artemis.statetransition.StateTransitionException;
 
 public class SlotProcessorUtil {
-  public static void updateLatestRandaoMixes(BeaconState state) {
-    // TODO: change values to UnsignedLong
```

Comment:
Why did this go away?  Was this intentional?  If so, I get this error after running: `./gradlew mash`

```
> Task :ethereum:statetransition:compileTestJava FAILED
/Users/jonny/projects/consensys/pegasys/artemis/ethereum/statetransition/src/test/java/tech/pegasys/artemis/statetransition/StateTransitionTest.java:74: error: 
cannot find symbol
    SlotProcessorUtil.updateLatestRandaoMixes(state);
                     ^
  symbol:   method updateLatestRandaoMixes(BeaconState)
  location: class SlotProcessorUtil
1 error
```

Reply:
Yes intentional. It's not in the spec anymore.